scripts/install_backend_python_libs.py

```diff
@@ -29,19 +30,53 @@
 
 
 def _normalize_python_library_name(library_name):
-    """Returns a normalized (lowercase) version of the python library name.
-
-    Python library name strings are case insensitive which means that
-    libraries are equivalent even if the casing of the library names are
-    different. This function normalizes python library names such that
-    identical libraries have the same library name strings.
+    """Returns a normalized version of the python library name.
+
+    Normalization of a library name means converting the library name to
+    lowercase, and removing any "[...]" suffixes that occur. The reason we do
+    this is because of 2 potential confusions when comparing library names that
+    will cause this script to find incorrect mismatches.
+
+    1. Python library name strings are case insensitive which means that
+       libraries are equivalent even if the casing of the library names are
```

Comment:
equivalent --> considered equivalent

names are --> names is (because you are referring to "the casing").